src/pnpq/devices/waveplate_thorlabs_kb10crm.py

```diff
@@ -61,7 +64,9 @@ def __init__(
         self.home_timeout = 20
         self.max_channel = 1
         self.auto_update = False
+        self.hub = check_usb_hub_connected()
```

Comment:
should be `self.hub_connected`. `self.hub` looks this represents a usb hub instance.